tests/workspace.json

```diff
@@ -1,7 +1,7 @@
 {
     "locale": {
         "plugins": [
-            "${PLUGINS}v4.8.0/coordinates.js"
+            "${PLUGINS}/v4.8.0/coordinates.js"
```

Comment:
Does this need to change in this PR?
Isn't this in line with the xyz_settings?

Reply:
No it doesn't need to change in this PR. In fact there should be an issue to review whether a workspace with private variables should be in the repository.